docs/content/v2025.1/yugabyte-platform/anywhere-automation/yb-kubernetes-operator.md

```diff
@@ -273,6 +273,226 @@ spec:
+
+**Setup**
+
+Set up incremental backups as follows:
+
+1. Apply the latest CRD for backup:
+
+    ```sh
+    kubectl apply -f https://raw.github.com/yugabyte/charts/{{< yb-version version="stable" format="short">}}/crds/concatenated_crd.yaml
+    ```
+
+1. Verify incremental backup fields in the backup CRD specification using `kubectl explain` to understand the available configuration options.
+
+    ```sh
+    $ kubectl explain backups.operator.yugabyte.io.spec
+    ```
+
+    ```output
+    GROUP: operator.yugabyte.io
+    KIND: Backup
+    VERSION: v1alpha1
+    FIELDS:
+    ...
+    incrementalBackupBase <string>
+      Base backup Custom Resource name. Operator will add an incremental backup to the existing chain of backups at the last.
+    ```
+
+##### Example
+
+This example decribes how to create and delete incremental backups, and assumes you have the following:
+
+- An existing YugabyteDB Anywhwere universe deployed using the YugabyteDB Kubernetes Operator.
+- A configured storage location for your backups. You must have an existing base backup (either a full backup or a previous incremental backup) to create the new incremental backup.
+
+Use the following CRD to create an incremental backup:
+
+```sh
+kubectl apply operator-backup-demo.yaml -n schedule-cr
+```
+
+```yaml
+#operator-backup-demo.yaml
+apiVersion: operator.yugabyte.io/v1alpha1
+kind: Backup
+metadata:
+  name: operator-backup-1
+spec:
+  backupType: PGSQL_TABLE_TYPE
+  storageConfig: az-config-operator-1
+  universe: operator-universe-test-1
+  timeBeforeDelete: 1234567890
+  keyspace: test
+  incrementalBackupBase: <base full backup cr name>
+  ```
+
+Deleting full backup deletes all incremental backups associated with it as follows:
+
+```sh
+# Get all backups in the 'schedule-cr' namespace.
+$ kubectl get backups -n schedule-cr
+```
+
+```output
+NAME                                                                     AGE
+operator-scheduled-backup-1-1069296176-full-2025-02-27-06-43-25          32m
+operator-scheduled-backup-1-1069296176-incremental-2025-02-27-06-59-26   16m
+operator-scheduled-backup-1-1069296176-incremental-2025-02-27-07-13-26   2m55s
+```
+
+```sh
+$ kubectl delete backup operator-scheduled-backup-1-1069296176-full-2025-02-27-06-43-25 -n schedule-cr
+```
+
+```sh
+$ kubectl get backups -n schedule-cr
+```
+
+```output
+No resources found in schedule-cr namespace.
+```
+
+#### Scheduled backups
+
+This feature is {{<tags/feature/ea>}}. Backup schedules support taking full backups based on cron expressions or specified frequencies. They also allow you to configure incremental backups to run in between these full backups, providing finer-grained recovery points.
+
+When an operator schedule triggers a backup, you need to create a corresponding CR for that specific backup. The operator names this CR appropriately, and marks it with ignore-reconciler-add.
```

Comment:
They dont need to create the CR. The CR for the automated backup will be created automatically and applied.
They only need to create the scheduled backups CR. The backups created from the schedule will auto-create their CRs.

Reply:
got it. I have modified based on your comments. Can you PTAL @kv83821-yb ?